tokio-console/src/config.rs

```diff
@@ -48,6 +50,22 @@ pub struct Config {
     #[clap(long = "log", env = "RUST_LOG")]
     log_filter: Option<LogFilter>,
 
+    /// Enable or disable specific linters.
+    ///
+    /// This is a comma-separated list of linters to enable or disable.
+    ///
+    /// Each linter is specified by its name, which is one of:
+    ///
+    /// * `self-wake-percent` -- Warns when a task wakes itself more than a certain percentage of its total wakeups.
+    ///
+    /// * `lost-waker` -- Warns when a task is dropped without being woken.
+    ///
+    /// * `never-yielded` -- Warns when a task has never yielded.
+    ///
+    /// [default: self-wake-percent, lost-waker, never-yielded]
+    #[clap(long = "linters")]
+    pub(crate) linters: Vec<KnownWarnings>,
```

Comment:
One other part of this issue that would be really great is the ability to tune the lint parameters. For example, the self wakes warning and the never yielded warning both have thresholds which could be configured.

Before we decide on how the command line arguments should be, it would be great to see if there's a way to model this iteratively.

For example, could we have something like: `--lint never-yielded=2s` to set the threshold to 2 seconds instead of the default 1 second for that lint?

Reply:
OK. Agreed, let's go ahead with this PR and consider how to configure warning parameters later on.